ACCWebAPI/Controllers/IncomingInvoiceController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using ACC.ViewModel.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using ACCDataModel.Kostenrechnung;
+
+namespace ACCWebAPI.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("incoming_invoices")]
+    public class IncomingInvoiceController : ControllerBase
+    {
+        private readonly ILogger<IncomingInvoiceController> _logger;
+        private readonly IACCService _accService;
+
+        public IncomingInvoiceController(ILogger<IncomingInvoiceController> logger, IACCService accService)
+        {
+            _logger = logger;
+            _accService = accService;
+        }
+
+        [HttpGet("")]
+        public async Task<IActionResult> Index()
```

Comment:
Lets rename this method to meet the .Net naming guidelines: GetIncomingInvoices()